src/sentry/incidents/subscription_processor.py

```diff
@@ -290,31 +290,37 @@ def get_crash_rate_alert_metrics_aggregation_value(self, subscription_update):
         count is just ignored
         - `crashed` represents the total sessions or user counts that crashed.
         """
-        session_status = tag_key(self.subscription.project.organization.id, "session.status")
-        data = {}
-
-        # ToDo(ahmed): Refactor this logic by calling the aggregate_query_results on the
-        #  BaseMetricsEntitySubscription cls
-        for row in subscription_update["values"]["data"]:
-            tag_value = reverse_tag_value(
-                self.subscription.project.organization.id, row[session_status]
-            )
-            data[tag_value] = row["value"]
+        try:
+            session_status = tag_key(self.subscription.project.organization.id, "session.status")
+            data = {}
+
+            # ToDo(ahmed): Refactor this logic by calling the aggregate_query_results on the
+            #  BaseMetricsEntitySubscription cls
+            for row in subscription_update["values"]["data"]:
+                tag_value = reverse_tag_value(
+                    self.subscription.project.organization.id, row[session_status]
+                )
+                data[tag_value] = row["value"]
 
-        total_session_count = data.get("init", 0)
-        crash_count = data.get("crashed", 0)
+            total_session_count = data.get("init", 0)
+            crash_count = data.get("crashed", 0)
 
-        if total_session_count == 0:
-            metrics.incr("incidents.alert_rules.ignore_update_no_session_data")
-            return
-
-        if CRASH_RATE_ALERT_MINIMUM_THRESHOLD is not None:
-            min_threshold = int(CRASH_RATE_ALERT_MINIMUM_THRESHOLD)
-            if total_session_count < min_threshold:
-                metrics.incr("incidents.alert_rules.ignore_update_count_lower_than_min_threshold")
+            if total_session_count == 0:
+                metrics.incr("incidents.alert_rules.ignore_update_no_session_data")
                 return
 
-        aggregation_value = round((1 - crash_count / total_session_count) * 100, 3)
+            if CRASH_RATE_ALERT_MINIMUM_THRESHOLD is not None:
+                min_threshold = int(CRASH_RATE_ALERT_MINIMUM_THRESHOLD)
+                if total_session_count < min_threshold:
+                    metrics.incr(
+                        "incidents.alert_rules.ignore_update_count_lower_than_min_threshold"
+                    )
+                    return
+
+            aggregation_value = round((1 - crash_count / total_session_count) * 100, 3)
+        except MetricIndexNotFound:
```

Comment:
Is this an expected/common case? If not, it might be worth logging the exception as well.